openarena-q3e

```diff
@@ -1,29 +1,59 @@
 #!/bin/sh
-OA_CVARS="+set fs_basegame baseoa +set protocol 71"
-DISABLED_CVARS="+set com_protocol 72 +set cl_voip 1"
 
+###
+CONFIG_FILE=".var/app/io.github.ec_.Quake3e.OpenArena/config/cfg"
+# Check if the config file exists
+if [ -f "$CONFIG_FILE" ]; then
+    source "$CONFIG_FILE"
+fi
+# Check if it's the first run or if RESPECT_XDGDIRS is not set
+if [ ! -f "$CONFIG_FILE" ] || [ -z "$RESPECT_XDGDIRS" ]; then
+    RESPECT_XDGDIRS=0
+
+    # Save RESPECT_XDGDIRS in the config file
+    echo "RESPECT_XDGDIRS=$RESPECT_XDGDIRS" > "$CONFIG_FILE"
+fi
+# Use the value of RESPECT_XDGDIRS to determine the directory
+if [ "$RESPECT_XDGDIRS" -eq 1 ]; then
+    # Use ~/.var/app/io.github.ec_.Quake3e.OpenArena/.openarena/ directory
+    mkdir -p "$HOME/.var/app/io.github.ec_.Quake3e.OpenArena/homepath"
+    OA_DIR="$HOME/.var/app/io.github.ec_.Quake3e.OpenArena/homepath"
+    rm -rf "$HOME/.openarena"
```

Comment:
Why removing the default directory?

Reply:
I don't like my home directory having tons of hidden folders

Reply:
It's inside the Flatpak directory, not directly in your home directory. Also, you are making this Flatpak for other users, not just for yourself.

Reply:
I don't like my home directory having tons of hidden folders. That was the purpose of this change

Reply:
> I don't like my home directory having tons of hidden folders. That was the purpose of this change

Your home directory will still have exactly the same amount of hidden folders since the `.openarena` dir is inside the Flatpak, not directly in you home directory.

> Also, why use .openarena if the user choose to use the other folder

As I already mentioned, removing a data directory that potentially can contain important user data (tweaked configs, maps and other custom pk3s etc.) without any warning is not a good solution that we can push to our users. We should at least move its content to the new directory before removing it.